src/services/url.ts

```diff
@@ -0,0 +1,37 @@
+import { IRouteKey, ROUTES } from '../constants';
+
+export function param(params: Record<string, string> = {}) {
```

Comment:
You can flush this out more as needed, but this app isn't using any complex query string values, so doing a manual deparam/param methods so as to reduce the list of dependencies